### ISSUE TITLE
inline task not working on 3.3.6

### ISSUE THREAD

**Issue by ZergRushJoe** [Netflix/conductor#2627]
**Describe the bug**
when running the system task "INLINE_TASK" getting error of 

`Empty 'evaluatorType' in Inline task's input parameters. A non-empty String value must be provided.`

I tried moving the evaluatorType from inside the inputParameters to the outside but get the same error

**Details**
Conductor version: 3.6
Persistence implementation: Postgres
Queue implementation: Postgres
Lock: none
Workflow definition: 
```
  {
    "name": "test",
    "description": "test",
    "version": 1,
    "schemaVersion": 2,
    "ownerEmail": "jkarp@uwm.com",
    "tasks": [
        {
            "name": "INLINE_TASK",
            "taskReferenceName": "inline_test",
            "type": "INLINE",
            "evaluatorType": "javascript",
            "inputParameters": {
                "inlineValue": "1",
                "evaluatorType": "javascript",
                "expression": "function scriptFun(){if ($.inlineValue == 1){ return {testvalue: true} } else { return {testvalue: false} }} scriptFun();"
            }
        }
    ]
  }
```

**To Reproduce**
1. run given workflow

**Expected behavior**
workflow to complete.

**Screenshots**
![image](https://user-images.githubusercontent.com/7153267/145104145-aafa3c63-bb48-4076-a54d-1c46e848f879.png)

**Comment by jxu-nflx** [comment]
Hello @ZergRushJoe I tried the exact workflow def that you posted, and it works fine for me. Can you check your workflow definition in UI to make sure it is correct?

**Comment by ZergRushJoe** [comment]
@jxu-nflx sorry for the wait. I copied the workflow I have above with the same result 
here is the copy from conductor itself:
```
{
  "updateTime": 1644327614860,
  "name": "inline_test",
  "description": "test",
  "version": 1,
  "tasks": [
    {
      "name": "INLINE_TASK",
      "taskReferenceName": "inline_test",
      "inputParameters": {
        "inlineValue": "1",
        "evaluatorType": "javascript",
        "expression": "function scriptFun(){if ($.inlineValue == 1){ return {testvalue: true} } else { return {testvalue: false} }} scriptFun();"
      },
      "type": "INLINE",
      "decisionCases": {},
      "defaultCase": [],
      "forkTasks": [],
      "startDelay": 0,
      "joinOn": [],
      "optional": false,
      "defaultExclusiveJoinTask": [],
      "asyncComplete": false,
      "loopOver": [],
      "evaluatorType": "javascript"
    }
  ],
  "inputParameters": [],
  "outputParameters": {},
  "schemaVersion": 2,
  "restartable": true,
  "workflowStatusListenerEnabled": false,
  "ownerEmail": "jkarp@uwm.com",
  "timeoutPolicy": "ALERT_ONLY",
  "timeoutSeconds": 0,
  "variables": {},
  "inputTemplate": {}
}
```

**Comment by ZergRushJoe** [comment]
I have closed this because I have found another way of doing what I needed

**Comment by ZergRushJoe** [comment]
@korolkevich7 seem to be fixed in 3.7 of conductor

**Comment by aravindanr** [comment]
Please refer the documentation, https://conductor.netflix.com/reference-docs/inline-task.html. The `evaluatorType` is a required field.
